src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.c

```diff
@@ -103,296 +103,296 @@ void EnSyatekiNiw_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80B11A94(EnSyatekiNiw* this, PlayState* play, s16 arg2) {
-    if (this->unk_254 == 0) {
-        if (arg2 == 0) {
-            this->unk_264 = 0.0f;
+void EnSyatekiNiw_UpdateAnimationTarget(EnSyatekiNiw* this, PlayState* play, s16 aniType) {
```

Comment:
```suggestion
void EnSyatekiNiw_UpdateAnimationTarget(EnSyatekiNiw* this, PlayState* play, s16 animationType) {
```

also this function doesn't just update the limb rot targets (every time timer0/timer2 reaches 0, btw these timers can probably be named after the relevant limbs), it also steps the rotations towards their respective targets

Overall this stuff may be a lot easier to name if we just figure out what the limbs 7/11/13 are
You can use https://github.com/zeldaret/Z64Utils to investigate assets
or the blender addon https://github.com/Fast-64/fast64 to import stuff

For example me using fast64 to look at gCuccoSkel + gCuccoAnim (I'm on Linux so no Z64Utils for me) :
![image](https://user-images.githubusercontent.com/5306619/206867206-06839d1d-0eb9-47c5-b452-2c6d0706b6f6.png)

Note names like `gCuccoLeftWingLimb` come from existing documentation, I didn't realize it but the cucco object object_niw is already documented ( https://github.com/zeldaret/oot/blob/master/assets/xml/objects/object_niw.xml )

There just hasn't been an enum made like for other skeletons, like for example `IMPA_LIMB_`... among others